modules/drivers/lidar/lidar_robosense/BUILD

```diff
@@ -0,0 +1,32 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
```

Comment:
Please rename the path from `lidar_robosense` to `robosense`.

Reply:
there is already have a robosense

Reply:
All LiDAR drivers for Robosense should be under the directory `robosense`, whether it be 16 channel or 128 channel.

Reply:
the new driver is another version of robosense which is more stable than current version. the architechure is different.So I was requested to upload this version by HU kuang. The folder "lidar_robosense "  was named by WANG xitong.
You can unify your opinions first.

Reply:
It's best practice to put all LiDAR drivers of a specific brand under the same top-level directory, and reduce as much code duplication as possible. A consistent and evolving component  is crucial for an open source project like Apollo since it will make the project more readable and maintainable. I think you would also agree with the phrase: `garbage in, garbage out`.

Reply:
I only agree with the phrase 'Liu Jiaming in Liu Jiaming out'

Reply:
你们内部需求都没对齐，先对齐需求再来找我吧。你不太擅长沟通，有什么事还是让王希同找我吧